src/main/java/org/usfirst/frc/team1294/robot/RobotMap.java

```diff
@@ -10,4 +11,7 @@
     // following variables to use with your drivetrain subsystem.
     public static int leftMotor = 2;
```

Comment:
Wait, isn't the robot wired so that left has a CAN id of 1, and right has a CAN id of 2?